### PR TITLE
Simplify some code

### PR DESCRIPTION
**Pull request by chefdd**

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@chefdd Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-boot&pullRequestId=17837)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-boot&pullRequestId=17837) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@chefdd Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-boot&pullRequestId=17837)!

**Comment by chefdd** [comment]
Simplify some code

File: spring-boot-project/spring-boot-devtools/src/test/java/org/springframework/boot/devtools/livereload/ConnectionInputStreamTests.java [hunk 1]
```diff
@@ -37,13 +37,15 @@
 	private static final byte[] NO_BYTES = {};
 
 	@Test
-	void readHeader() throws Exception {
-		String header = "";
+	public void readHeader() throws Exception {
```
Comment (snicoll) [inline spring-boot-project/spring-boot-devtools/src/test/java/org/springframework/boot/devtools/livereload/ConnectionInputStreamTests.java]: A Junit Jupiter test doesn't have to be public.

Comment (chefdd) [reply spring-boot-project/spring-boot-devtools/src/test/java/org/springframework/boot/devtools/livereload/ConnectionInputStreamTests.java]: > A Junit Jupiter test doesn't have to be public.

ok,thanks

**Comment by snicoll** [comment]
>  please fix that and run the Maven build locally.

@chefdd thanks for the update but I think the build is still failing. Can you please build locally before pushing?

**Comment by chefdd** [comment]
> > please fix that and run the Maven build locally.
> 
> @chefdd thanks for the update but I think the build is still failing. Can you please build locally before pushing?

sorry , I will double check and then submit

**Comment by snicoll** [comment]
@checketts thank you for making your first contribution to Spring Boot. In the future, please run the build locally before submitting your proposal as each commit sends a notification to a large number of watchers. If you have several commits, [squashing them in a single commit](https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git) will be appreciated as we'd have to do that when reviewing your submission anyway.

**Pull request closed without merging**